snyk-operator/deploy/olm-catalog/snyk-operator/0.0.0/snyk-operator.v0.0.0.clusterserviceversion.yaml

```diff
@@ -103,6 +114,30 @@ spec:
             path: image.tag
             x-descriptors:
             - 'urn:alm:descriptor:text'
+          - description: >-
+              True to use a PVC for temporary storage, false to use emptyDir
```

Comment:
Just to note we should probably add a full stop to all of these descriptions since they're shown in the UI